.azurepipelines/MuDevOpsWrapper.yml

```diff
@@ -17,7 +17,7 @@ resources:
       type: github
       endpoint: microsoft
       name: microsoft/mu_devops
-      ref: refs/tags/v6.1.0
+      ref: refs/tags/v6.4.1

```

Comment:
Can you please go ahead and update to 6.5.1? Since that has some related changes we can double check everything is good there as well.

Reply:
Yeah, I was thinking about it. But did I miss something? I thought this file will be synced from mu_devops. But it seems that the sync PRs never arrived?

Reply:
Hmm, all cargo command step start to fail: https://dev.azure.com/projectmu/mu/_build/results?buildId=54925&view=logs&j=b1ece871-faff-54b3-ec9d-bb6ab43926f1&t=473d1ea5-21b8-59d1-1993-78db6d565df4. Do you know why that might be?

Reply:
> Hmm, all cargo command step start to fail: https://dev.azure.com/projectmu/mu/_build/results?buildId=54925&view=logs&j=b1ece871-faff-54b3-ec9d-bb6ab43926f1&t=473d1ea5-21b8-59d1-1993-78db6d565df4. Do you know why that might be?

There's another change you need to pass the `container_build` parameter:

https://github.com/microsoft/mu_devops/blob/main/.sync/azure_pipelines/MuDevOpsWrapper.yml#L118-L120

```yaml
    - ${{ parameters.extra_cargo_steps }}
    - template: Steps/RustCargoSteps.yml@mu_devops
      parameters:
        container_build: true
```

Reply:
Aye, just found that and added. Let's see how it goes.

Reply:
Last run on 6.5.1 seems good. Updated this branch to top of main. Will merge once all checks pass.